beacon-chain/rpc/beaconv1/state_test.go

```diff
@@ -29,12 +39,12 @@ func TestGetGenesis(t *testing.T) {
 			GenesisTimeFetcher: chainService,
 			ChainInfoFetcher:   chainService,
 		}
-		resp, err := s.GetGenesis(context.Background(), &ptypes.Empty{})
+		resp, err := s.GetGenesis(ctx, &ptypes.Empty{})
 		require.NoError(t, err)
-		assert.Equal(t, genesis.Unix(), resp.GenesisTime.Seconds)
-		assert.Equal(t, int32(0), resp.GenesisTime.Nanos)
-		assert.DeepEqual(t, validatorsRoot[:], resp.GenesisValidatorsRoot)
-		assert.DeepEqual(t, []byte("genesis"), resp.GenesisForkVersion)
+		assert.Equal(t, genesis.Unix(), resp.Data.GenesisTime.Seconds)
+		assert.Equal(t, int32(0), resp.Data.GenesisTime.Nanos)
+		assert.DeepEqual(t, validatorsRoot[:], resp.Data.GenesisValidatorsRoot)
+		assert.DeepEqual(t, []byte("genesis"), resp.Data.GenesisForkVersion)
```

Comment:
Diff in tests of another API function is caused by an updated ethereumapis dependency.